r_basics_introduction/r_basics_introduction.md

```diff
@@ -0,0 +1,784 @@
+
+The first is the **header** which includes information about the document, such as its title, author, and the desired output format when the document is rendered. A lot of this information is optional so a header can be pretty short, as this one here, or much longer.
+
+The second type of building block is **text**. Text can include special kinds of marks (Markdown) that add styling.  This includes hash marks that turn a line of text into a header, asterisks that can create italics, lines of dashes that turn into horizontal lines, and more.
+
+The third is **code chunks**.  Code chunks contain R code that can be executed to output results.
+
+So how do you execute the code in the code chunks?  There are a couple of ways.
+
+The first is to run the code in a single code chunk. And you do this by clicking the green right-pointing triangle on the top right of the chunk. This symbol looks like a "play" button.  There are other symbols, too, including one that means "run all the code chunks that came before this one".
+
+![A code chunk with a green play button indicated](media/run_chunk.png)<!-- style = "max-width: 600px;" -->
+
+When you click the play button in a chunk, as shown above, the code will execute.  Then, immediately below the code chunk, you will see any results of the code printed on your screen.  In this case, you would see some statistics about a normal distribution.
+
+The second way to execute code is to convert the entire R Markdown document into HTML (or another target format).  For now we'll stick to HTML because it's the default and usually what you want. This process of converting R Markdown into a target format is called "rendering" or "knitting" – the mental model here is that you're knitting together marked up text, code, and results into a nicely formatted document that you can then present or share with others.
+
+To **knit** a document, click the button that says "Knit" next to a blue ball of yarn on the top of the Editor pane.  Then in a few moments, a rendered document will be shown:
+
+![RStudio window with the knit button indicated](media/knit_button.png)<!-- style = "max-width:600px;"-->
+![Result of knitting, the start of an html document with headers and code](media/rendered_markdown.png)<!-- style = "max-width:600px;"-->
+
+### R Markdown in Detail
+
+Let's look at this R Markdown document in a little more detail.  Below is an R Markdown document and the resulting output in HTML that was created when this R Markdown document was knitted.
+
+Below these images, we'll explain what each part of the R Markdown includes in a bit more detail.
+
+<div style = "align-items: center; display: flex;">
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+![25 lines of R Markdown](media/r_markdown_detail.png)
+</div>
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+
+![A brief report with headers, some statistics, and a graph](media/rendered_detail.png)
```

Comment:
This might be too wordy but I replaced with the following.  LMK what you think!

<div style = "align-items: center; display: flex;">
<div style = "margin: 1rem; max-width: 50%; float:left;">
![R Markdown code.  It begins with a short header enclosed by lines containing three dashes. Then text with a few special markdown characters alternates with code chunks.  Code chunks are preceded by a line with three backticks and the letter r in curly braces, and are followed by a line with three backticks. ](media/r_markdown_detail.png)
</div>
<div style = "margin: 1rem; max-width: 50%; float:left;">

![A brief report that is created when the R Markdown code is rendered.  The R Markdown header appears in the form of a large title.  The R Markdown text appears as appropriately rendered text with bold, italics, and other decorations.  The code chunks are represented by code and code output, such as a graph.](media/rendered_detail.png)
</div>
</div>

Reply:
made some suggestions

Reply:
included!